.travis.yml

```diff
@@ -22,15 +31,29 @@ matrix:
       env: DB=MYSQL CORE_RELEASE=3.1
     - php: 5.6
       env: DB=PGSQL CORE_RELEASE=3.2
-  allow_failures:
-    - php: 7.0
```

Comment:
It's still there, see https://github.com/gordonbanderson/silverstripe-iframe/blob/88dbe458302063dd6672b7575357cc385a0a0385/.travis.yml